controllers/admin/AdminSuppliersController.php

```diff
@@ -67,6 +67,8 @@ public function __construct()
             'products' => array('title' => $this->trans('Number of products', array(), 'Admin.Catalog.Feature'), 'align' => 'right', 'filter_type' => 'int', 'tmpTableFilter' => true),
             'active' => array('title' => $this->trans('Enabled', array(), 'Admin.Global'), 'align' => 'center', 'active' => 'status', 'type' => 'bool', 'orderby' => false, 'class' => 'fixed-width-xs'),
         );
+
+        $this->addTopAlertMessage($this->supplierInfoMessage());
```

Comment:
This message is added on a legacy page, but maybe it's gonna be migrated during 177 development. We should update the migration issue so that this feature is not forgotten.

Reply:
Oh actually the listing page has already been migrated but is hidden for now: https://github.com/PrestaShop/PrestaShop/issues/10545
You can access it by typing the url directly, so you should modify the migrated page rather than the legacy one

Reply:
yep, done !